### PR TITLE
(GH-5) Migrate to netstandard2.0

### LINKED ISSUE #5: Migrate to .NET Standard
Addin should support .NET Core and Cake CoreCLR runner

### PR DESCRIPTION
**Pull request by vhatsura**
Fixes #5

### CODE REVIEW

File: setup.cake [hunk 1]
```diff
@@ -16,9 +16,9 @@ BuildParameters.PrintParameters(Context);
 
 ToolSettings.SetToolSettings(
     context: Context,
-    dupFinderExcludePattern: new string[] { BuildParameters.RootDirectoryPath + "/src/Cake.Issues.Reporting.Tests/*.cs" },
+    dupFinderExcludePattern: new string[] { BuildParameters.RootDirectoryPath + "/src/Cake.Issues.Reporting.Tests/*.cs", BuildParameters.RootDirectoryPath + "/src/Cake.Issues.MsBuild*/**/*.AssemblyInfo.cs" },
```
Comment (pascalberger) [inline setup.cake]: Path seems to be wrong

File: src/Cake.Issues.Reporting.Tests/Cake.Issues.Reporting.Tests.csproj [hunk 2]
```diff
@@ -1,114 +1,34 @@
...
-    <CodeAnalysisRuleSet>..\Cake.Issues.Reporting.Tests.ruleset</CodeAnalysisRuleSet>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRuleSet>..\Cake.Issues.Reporting.Tests.ruleset</CodeAnalysisRuleSet>
```
Comment (pascalberger) [inline src/Cake.Issues.Reporting.Tests/Cake.Issues.Reporting.Tests.csproj]: Ruleset missing in the new csproj

File: setup.cake (hunk 1 above)
Comment (vhatsura) [reply setup.cake]: Fixed

File: src/Cake.Issues.Reporting.Tests/Cake.Issues.Reporting.Tests.csproj (hunk 2 above)
Comment (vhatsura) [reply src/Cake.Issues.Reporting.Tests/Cake.Issues.Reporting.Tests.csproj]: Fixed

**Comment by vhatsura** [comment]
@pascalberger, all comments were fixed

**Comment by pascalberger** [comment]
@vhatsura your changes have been merged, thanks for your contribution 👍

**Pull request merged**